js/ui/windowManager.js

```diff
@@ -1458,7 +1456,7 @@ const WindowManager = new Lang.Class({
             }
         }
 
-        if (this._codingManager) {
+        if (global.settings.get_boolean('enable-behind-the-screen')) {
```

Comment:
Good catch. The other short-term solution would be to require a restart to disable/enable the feature. Otherwise we would need to listen on the gsetting change and add a cleanup function that iters over the existing coding sessions and clean those.